CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2022-02-15
+| What's new | |
+| :-- | --- |
+| You can diff against and run subgraph checks on your registered graphs with **subgraphs in Sandbox**. Check out how [here](https://www.apollographql.com/docs/studio/explorer/sandbox/#subgraph-checks).| ![subgraph-sandbox-one-check-failed](https://user-images.githubusercontent.com/14367451/154148356-2fabcd82-1124-4a3b-bd70-5820e33f7895.jpg)
```

Comment:
Can we instead say the following:
"Today we introduced the ability to work with your subgraphs locally in Sandbox! If the endpoint that you add to Sandbox points to a subgraph, a `SUBGRAPH` badge will appear. You can then run operations against your local subgraph schema, diff your changes against a registered subgraph schema and run checks to ensure that your changes do not affect composition or any running operations to the corresponding federated graph. Learn more [here](https://www.apollographql.com/docs/studio/explorer/sandbox/#subgraph-checks)." 

I think we also want to add an image of Sandbox that points to Subgraph badge